Patches/Rimworld-Style Pilas And Bows Strapped With Grenades and Shells Extended/Javelin.xml

```diff
@@ -0,0 +1,707 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Rimworld-Style Pilas And Bows Strapped With Grenades and Shells Extended</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+		<operations>
+
+            <!--PILA/PILUM/JAVELIN-->
+            <!--Add the ammo to the AmmoSet (for javelin/Pila launchers like atlatl)-->
+			<!--li Class="PatchOperationAdd">
+				<xpath>/Defs/CombatExtended.AmmoSetDef[defName="AmmoSet_Javelins"]/ammoTypes</xpath>
+				<value>
+		        	<PilaHighExplosive>PilumHE_Fired</PilaHighExplosive>
+		        	<PilaHighIncendiary>PilumHI_Fired</PilaHighIncendiary>
+		        	<PilaEMP>PilumEMP_Fired</PilaEMP>
+		        	<PilaSmoke>PilumSmoke_Fired</PilaSmoke>
+		        	<PilaFirefoam>PilumFirefoam_Fired</PilaFirefoam>
+		        	<PilaAntigrain>PilumAntigrain_Fired</PilaAntigrain>
+                </value>
+            </li-->
+
+            <!--Overwrite the Pilas added by the mod-->
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs</xpath>
+				<value>
+
+                    <ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseWeaponAndAmmoNeolithic" Name="PilaStrappedBased">
+                        <defName>PilaHighExplosive</defName>
+                        <label>HEla</label>
+                        <description>This shell won't miss 9 tiles away from its target (It might miss 2 but that's still valid).</description>
+                        <graphicData>
+                            <texPath>HEla</texPath>
+                            <graphicClass>Graphic_Single</graphicClass>
+                        </graphicData>
+                        <soundInteract>Interact_BeatFire</soundInteract>
+                        <statBases>
+                            <MarketValue>62.35</MarketValue>
+                            <SightsEfficiency>0.45</SightsEfficiency>
+                            <ShotSpread>1.5</ShotSpread>
+                            <SwayFactor>2.5</SwayFactor>
+                            <Bulk>3.5</Bulk>
+                            <Mass>2.00</Mass>
+                            <RangedWeapon_Cooldown>1</RangedWeapon_Cooldown>
+                        </statBases>
+                        <!--equippedAngleOffset>30</equippedAngleOffset-->
+                        <stackLimit>25</stackLimit>
+                        <weaponTags>
+                            <li>NeolithicRangedHeavy</li>
+                            <li>CE_Pila</li>
+                            <li>CE_OneHandedWeapon</li>
+                            <!--li>IndustrialRangedHeavy</li> 
+                            <li>GrenadeDestructive</li>
+                            <li>EmpireGrenadeDestructive</li--> <!-- Not sure whether to inherit the tags from the base mod, setting it to default for pila for now -->

```

Comment:
I would stick to whatever tags they get from the base mod (plus CE onehanded)--giving it default pila tags may cause oddities of tribals inexplicably showing up with explosive or EMP projectiles.